superset/views/base.py

```diff
@@ -61,6 +61,8 @@
 FRONTEND_CONF_KEYS = (
     "SUPERSET_WEBSERVER_TIMEOUT",
     "SUPERSET_DASHBOARD_POSITION_DATA_LIMIT",
+    "SUPERSET_DASHBOARD_PERIODICAL_REFRESH_LIMIT",
+    "SUPERSET_DASHBOARD_PERIODICAL_REFRESH_WARNING_MESSAGE",
```

Comment:
can you add these two config options to the config.py file too and document them there? You can default it to 0/None if you'd like, or provide a generic message, but i think they should be documented there regardless

Reply:
fixed.